Command/DoctrineCommand.php

```diff
@@ -32,7 +32,7 @@ public static function configureMigrations(ContainerInterface $container, Config
     {
         if (!$configuration->getMigrationsDirectory()) {
             $dir = $container->getParameter('doctrine_migrations.dir_name');
-            if (!@mkdir($dir, 0777, true) && !is_dir($dir)) {
+            if (!is_dir($dir) && !@mkdir($dir, 0777, true)) {
```

Comment:
You can do the check before and after the`mkdir()` call. Though I wonder why the call triggers an error at all while being silenced.